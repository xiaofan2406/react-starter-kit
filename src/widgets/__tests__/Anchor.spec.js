import React from 'react';
import { render } from 'react-testing-library';

import Anchor from '../Anchor';

test('Anchor matches snapshot', () => {
  const { container } = render(
    <Anchor href="https://github.com/xiaofan2406/react-starter-kit">
      link
    </Anchor>
  );

  expect(container.firstChild).toMatchSnapshot();
});
