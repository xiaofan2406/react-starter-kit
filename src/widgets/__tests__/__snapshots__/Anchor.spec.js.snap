// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Anchor matches snapshot 1`] = `
<a
  href="https://github.com/xiaofan2406/react-starter-kit"
  rel="noopener noreferrer"
  target="_blank"
>
  link
</a>
`;
